### PR TITLE
ODIM-3191 http handler of get/delete an aggregate in api service

### PR DESCRIPTION
**Pull request by irappa-pattar**

### CODE REVIEW

File: svc-api/handle/aggregator_test.go [hunk 1]
```diff
@@ -440,24 +438,23 @@ func TestDeleteAggregate(t *testing.T) {
 	redfishRoutes := testApp.Party("/redfish/v1/AggregationService/Aggregates/{id}")
 	redfishRoutes.Delete("/", a.DeleteAggregate)
 	test := httptest.New(t, testApp)
-	//  update status code after the code is added
 	// test with valid token
 	test.DELETE(
 		"/redfish/v1/AggregationService/Aggregates/7ff3bd97-c41c-5de0-937d-85d390691b73",
-	).WithHeader("X-Auth-Token", "ValidToken").Expect().Status(http.StatusNotImplemented)
+	).WithHeader("X-Auth-Token", "ValidToken").Expect().Status(http.StatusOK)
```
Comment (jeevan-kamkar) [inline svc-api/handle/aggregator_test.go]: StatusNoContent

**Review by jeevan-kamkar — CHANGES_REQUESTED** [review]
Address the requested changes

File: svc-api/handle/aggregator_test.go (hunk 1 above)
Comment (irappa-pattar) [reply svc-api/handle/aggregator_test.go]: updated

**Review by jeevan-kamkar — APPROVED** [review]
LGTM

**Review by Bharath-KKB — APPROVED** [review]

**Pull request merged**